index.bs

```diff
@@ -1670,8 +1676,8 @@ with the fields of the attestation certificate's extension data.
     - Verify that {{AndroidKeyAttestation/signature}} is a valid certificate chain, consisting of a time-valid X.509 certificate
         chaining up to a trusted attestation root key.
 
-    - Verify that the public key in the attestation certificate matches the credential public key in the attestation data field
-        of the given <a>authenticatorData</a>.
+    - Verify that the public key in the attestation certificate matches the <a>credential public key</a> in the attestation 
```

Comment:
Suggestion based on earlier email thread: "attestation certificate" could be replaced with something like "leaf certificate in the chain represented by {{AndroidKeyAttestation/signature}}".